### DISCUSSION TITLE
sidenotes in an arbitrary (or fixed) Jekyll theme

### DISCUSSION THREAD

**Discussion by abourque72** [kaushikgopal/henry-jekyll#14] [General]
Hi,

I use a different Jekyll theme (minima), and I'm not interested in changing theme. However, I am interested in Henry's sidenotes feature. Would it be possible to port this to other themes without too much effort?

**Comment by kaushikgopal** [comment]
it's definitely possible to port but it would require some effort/knowledge. 

i switched my own blog to use hugo and maintain henry for hugo now @ https://github.com/kaushikgopal/henry-hugo. so can't offer to do it myself.
